openstudio-workflow.gemspec

```diff
@@ -33,6 +33,6 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'rainbow', '~> 3.0.0'
   s.add_development_dependency 'rake', '~> 12.3'
   s.add_development_dependency 'rspec', '~> 3.9'
-  s.add_development_dependency 'rubocop', '~> 0.80.1'
+  s.add_development_dependency 'rubocop', '~> 0.54.0'
```

Comment:
not sure how we had this locked to 0.80.1. Downgrading to 0.54 which is what we are supporting with measure tester.